packages/components/src/SidePanel/SidePanel.module.scss

```diff
@@ -209,7 +209,7 @@ $large-docked-width: 7rem;
 		}
 	}
 
-	&.reverse[role='navigation'][aria-expanded] {
```

Comment:
do you know when we have loosed this expanded attribute ?

Reply:
Not at all 👀 
But this attribute check has not to be check here as we want the style to be apply regardless of the expanded or not status